zabbix-probes/zabbix-probes-common/src/main/java/com/indigo/zabbix/utils/KeystoneTokenProvider.java

```diff
@@ -16,6 +16,10 @@
   @RequestLine("GET /v3/OS-FEDERATION/identity_providers/indigo-dc/protocols/oidc/auth")
   @Headers("Authorization: Bearer {token}")
   Response getKeystoneToken(@Param("token") String accessToken);
+  
+  @RequestLine("GET /v3/OS-FEDERATION/identity_providers/indigo-dc/protocols/iamoidc/auth")
```

Comment:
I have it covered in one of my changes that I'll upload after you (I'll have to do some conflict solving) Basically you can (and I will) add the protocol as a parameter so you can rewrite the first method with an extra {protocol} element in the path where oidc is and then pass either oidc or iamoidc